src/Core/Resolvers/SqlQueryEngine.cs

```diff
@@ -327,16 +332,18 @@ public static OkObjectResult OkResponse(JsonElement jsonResult)
         // Unlike a normal query, result from database may not be JSON. Instead we treat output as SqlMutationEngine does (extract by row).
         // As such, this could feasibly be moved to the mutation engine. 
         // </summary>
-        private async Task<JsonDocument?> ExecuteAsync(SqlExecuteStructure structure)
+        private async Task<JsonDocument?> ExecuteAsync(SqlExecuteStructure structure, string dataSourceName = "")
```

Comment:
Do we need this defaulting to empty string here or other upstream method invocations if it is also defaulted downstream in executors?

Reply:
This will be changed in future pr's If i dont default this here, then have to change the calls which I did not want to do as of now. This will be removed in the next pr.